install.sh

```diff
@@ -27,9 +27,10 @@ if [ -n "$NAME" ] ; then
 fi
 
 mkdir -p "$HOST$DATADIR"
+rm -f "$HOST$DATADIR"/docker-run-opts
```

Comment:
this ensures docker-run-opts file is empty in case install script fails in between while file was already modified